src/contracts/index.ts

```diff
@@ -132,6 +133,13 @@ export class Contracts {
     });
   }
 
+  OldERC721Wrapper(address: Address) {
+    return new OldERC721Wrapper({
+      contracts: this,
```

Comment:
I don't love this. I feel like OldERC721Wrapper ends up doing things with 2 contracts. I would first do the transfer, then the wrap. Both of this calls would be on `gondi.ts wrapOldERC721` fn